src/main/resources/CodyBundle.properties

```diff
@@ -116,27 +116,22 @@ duration.x-months-ago={0} months ago
 duration.last-year=Last year
 duration.x-years-ago={0} years ago
 duration.x-ago={0} ago
-
 popup.select-chat=Select Chat
 popup.export-chat=Export Chat As JSON
 popup.remove-chat=Remove Chat
 popup.remove-all-chats=Remove All Chats
 export.failed=Cody: Chat export failed. Please retry...
 export.timed-out=Cody: Chat export timed out. Please retry...
-
 PromptPanel.ask-cody.message=Message (type @ to include specific files as context)
 PromptPanel.ask-cody.follow-up-message=Follow-up message (type @ to include specific files as context)
 LlmDropdown.disabled.text=Start a new chat to change the model
-
```

Comment:
no strong feeling but for me some spacing b/w the specific sections here make it more readable

Reply:
I have not removed it, it must be `spotless`.
I do not know how it could have been committed that way before, I guess we do not have proper formatting check on CI.

Reply:
It was intelliJ after all, 
`Editor > Code Style > Properties > "Keep blank lines"`